lib/matplotlib/animation.py

```diff
@@ -931,6 +934,11 @@ def __init__(self, fig, event_source=None, blit=False):
         if self._blit:
             self._setup_blit()
 
+    def __del__(self):
+        if not getattr(self, '_anim_was_inited', True):
+            warnings.warn('Animation was garbage-collected before saving any '
+                          'frames; did you forget to save it to a variable?')
```

Comment:
```suggestion
            warnings.warn(
                'Animation was deleted without rendering anything. This is '
                'most likely unintended. To prevent deletion, assign the '
                'Animation to a variable that exists as long as you need '
                'the Animation.')
```
Our users don't necessarily know what garbage collection is. Also let's be explicit what to do instead of a suggestive question.